<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AndroidSDRDriverInputGui</class>
 <widget class="QWidget" name="AndroidSDRDriverInputGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>181</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>181</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>181</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Remote TCP Input</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_freq">
     <property name="topMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="deviceUILayout">
       <item>
        <layout class="QHBoxLayout" name="deviceButtonsLayout">
         <item>
          <widget class="ButtonSwitch" name="startStop">
           <property name="toolTip">
            <string>start/stop acquisition</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../sdrgui/resources/res.qrc">
             <normaloff>:/play.png</normaloff>
             <normalon>:/stop.png</normalon>:/play.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="deviceRateLayout">
         <item>
          <widget class="QLabel" name="deviceRateText">
           <property name="toolTip">
            <string>Device I/Q sample rate kS/s</string>
           </property>
           <property name="text">
            <string>00000k</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="freqLeftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ValueDial" name="centerFrequency" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Center frequency in kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="freqUnits">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string> kHz</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_ppm">
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="ppmLabel">
       <property name="text">
        <string>LO ppm</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="ppm">
       <property name="toolTip">
        <string>Local Oscillator ppm correction</string>
       </property>
       <property name="minimum">
        <number>-200</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="ppmText">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_rateTime">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="gridLayout_corr">
     <item>
      <widget class="ButtonSwitch" name="dcOffset">
       <property name="toolTip">
        <string>DC Offset auto correction</string>
       </property>
       <property name="text">
        <string>DC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="iqImbalance">
       <property name="toolTip">
        <string>IQ Imbalance auto correction</string>
       </property>
       <property name="text">
        <string>IQ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="biasTee">
       <property name="toolTip">
        <string>Enable bias tee</string>
       </property>
       <property name="text">
        <string>T</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="directSampling">
       <property name="toolTip">
        <string>Direct sampling</string>
       </property>
       <property name="text">
        <string>DS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="sampleBitsLabel">
       <property name="text">
        <string>IQ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sampleBits">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Bit depth per I or Q sample transmitted over network</string>
       </property>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sampleBitsUnits">
       <property name="text">
        <string>bits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dataAddressLabel">
       <property name="minimumSize">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dataPortSeparator">
       <property name="text">
        <string>:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataPort">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Remote data port (rtl_tcp defaults to 1234)</string>
       </property>
       <property name="inputMask">
        <string>00000</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_address">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sampleRateLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="devSampleRateLabel">
       <property name="text">
        <string>SR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValueDial" name="devSampleRate" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Remote device sample rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="devSampleRateUnits">
       <property name="text">
        <string>S/s</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="rfBWLabel">
       <property name="text">
        <string>BW</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValueDial" name="rfBW" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>RF filter bandwidth (kHz)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rfBWUnits">
       <property name="text">
        <string>kHz</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="gainLayout">
     <item>
      <widget class="QLabel" name="gain1Label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Gain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="gain1">
       <property name="toolTip">
        <string>Gain</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gain1Text">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>40.0dB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="gainLine1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gain2Label">
       <property name="text">
        <string>Gain</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="gain2">
       <property name="toolTip">
        <string>Gain</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gain2Text">
       <property name="text">
        <string>20dB</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ButtonSwitch" name="agc">
       <property name="toolTip">
        <string>Toggle automatic gain control</string>
       </property>
       <property name="text">
        <string>AGC</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_rfBW">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
  <customwidget>
   <class>ValueDial</class>
   <extends>QWidget</extends>
   <header>gui/valuedial.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>startStop</tabstop>
  <tabstop>centerFrequency</tabstop>
  <tabstop>ppm</tabstop>
  <tabstop>dcOffset</tabstop>
  <tabstop>iqImbalance</tabstop>
  <tabstop>agc</tabstop>
 </tabstops>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
